### PR TITLE
If offboarded, do not allow client to submit

### PR DESCRIPTION
**Pull request by wschaefer**
Co-authored-by: Whitney Schaefer <wschaefer@codeforamerica.org>

### CODE REVIEW

File: spec/controllers/ctc/questions/simplified_filing_income_offboarding_controller_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 describe Ctc::Questions::SimplifiedFilingIncomeOffboardingController do
   describe '#show' do
-    let(:intake) { create :ctc_intake, client: build(:client, tax_returns: [build(:tax_return, filing_status: filing_status)]) }
+    let(:intake) { create :ctc_intake, client: build(:client, tax_returns: [build(:tax_return, filing_status: filing_status, year: TaxReturn.current_tax_year)]) }
```
Comment (alaroia-cfa) [inline spec/controllers/ctc/questions/simplified_filing_income_offboarding_controller_spec.rb:5]: Oh dear - what a subtle thing to change. Maybe it makes sense to change the factory. Not a blocker.

File: app/views/ctc/questions/confirm_legal/edit.html.erb [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <%= f.cfa_checkbox(:consented_to_legal, t("views.ctc.questions.confirm_legal.consent"), options: { checked_value: "yes", unchecked_value: "no" }) %>
     <%= recaptcha_v3(action: 'confirm_legal') %>
     <div class="spacing-above-60">
-      <%= f.continue t("views.ctc.questions.confirm_legal.action") %>
+      <%= f.submit(t("views.ctc.questions.confirm_legal.action"), class: "button button--primary button--wide", disabled: current_intake.benefits_eligibility.disqualified_for_simplified_filing?) %>
```
Comment (alaroia-cfa) [inline app/views/ctc/questions/confirm_legal/edit.html.erb:13]: Dumb request - should we also make the backend prevent people from submitting even if they monkey with the button to make it un-disabled? Not a blocker.

**Review by alaroia-cfa — APPROVED** [review]
I like it! Two non-blocker suggestions.

File: spec/controllers/ctc/questions/simplified_filing_income_offboarding_controller_spec.rb (hunk 1 above)
Comment (wschaefer) [reply spec/controllers/ctc/questions/simplified_filing_income_offboarding_controller_spec.rb:5]: l8r

**Pull request merged**